docs/commands/math.md

```diff
@@ -8,8 +8,10 @@ Currently the following functions are implemented:
 * `math min`: Finds the minimum within a list of numbers or tables
 * `math max`: Finds the maximum within a list of numbers or tables
 * `math median`: Finds the median of a list of numbers or tables
+* `math stddev`: Finds the standard deviation of a list of numbers or tables
 * `math sum`: Finds the sum of a list of numbers or tables
 * `math product`: Finds the product of a list of numbers or tables
+* `math variance`: Finds the variance of a list of numbers or tables
```

Comment:
While you're in here, if you don't mind, two small things:

1. Let's keep it sorted alphabetically, so could you tweak it to be max -> median -> min?
2. There's also `math mode`

Reply:
Of course, added both suggestions.